.github/workflows/ci.yaml

```diff
@@ -19,15 +19,36 @@ on:
       - "pull-request/[0-9]+"
       - main
       - release-*
-  
+
 jobs:
   basic:
     uses: ./.github/workflows/basic-checks.yaml
 
-  image:
-    uses: ./.github/workflows/image.yaml
-    needs:
-    - basic
-    secrets: inherit
-    with:
-      version: ${{ needs.basic.outputs.version }}
+  chart:
+    runs-on: linux-amd64-cpu4
+    container:
+      image: docker.io/alpine/helm
+    steps:
+      - uses: actions/checkout@v5
+        name: checkout
+      - name: build-and-push-chart
+        run: |
+          # use GNU sed
+          apk add --no-cache make diffutils sed
+          export VERSION="$(make print-VERSION)"
+          export VERSION_W_COMMIT="$(make print-VERSION)-$(echo $GITHUB_SHA | cut -c1-8)"
+          # remove leading 'v'
+          echo "replace ${VERSION:1} with ${VERSION_W_COMMIT:1}"
+          sed -i.bak "s/${VERSION:1}/${VERSION_W_COMMIT:1}/g" deployments/helm/nvidia-dra-driver-gpu/Chart.yaml
+          (cd deployments/helm/nvidia-dra-driver-gpu && diff -wu Chart.yaml.bak Chart.yaml) || echo "ignore diff exit code"
+          helm package deployments/helm/nvidia-dra-driver-gpu
+          echo ${{ secrets.GITHUB_TOKEN }} | helm registry login --password-stdin ghcr.io -u NVIDIA
+          helm push nvidia-dra-driver-gpu-*tgz oci://ghcr.io/NVIDIA/k8s-dra-driver-gpu
```

Comment:
Looking at the error message, I think we need to change the name of the chart to `k8s-dra-driver-gpu`.

Reply:
Looking good now.